js/config.js

```diff
@@ -37,7 +37,7 @@ let respecConfig = {
         companyURL: "https://logius.nl",
```

Comment:
Bij de authors moeten we overwegen om Frank en Niels er te zetten, want wij zijn editors. Misschien even vragen aan hen of zij daar oke mee zijn? Dit mag ook in een latere PR.